lib/Metrc/client.rb

```diff
@@ -5,6 +5,7 @@ module Metrc
   class Client
     include HTTParty
     headers 'Content-Type' => 'application/json'
+    NEW_PACKAGE_PLANTING_STATES = Set.new([:ca]).freeze
```

Comment:
if we're managing this constant ourselves, and only use it for `include?` there's no need to make it a Set.

A simpler `%i{ca}.freeze` or `[:ca].freeze` makes it more legible.